go/scheduler/tests/tester.go

```diff
@@ -23,8 +23,8 @@ const recvTimeout = 5 * time.Second
 
 // SchedulerImplementationTests exercises the basic functionality of a
 // scheduler backend.
-func SchedulerImplementationTests(t *testing.T, backend api.Backend, consensus consensusAPI.Backend) {
-	seed := []byte("SchedulerImplementationTests")
+func SchedulerImplementationTests(t *testing.T, name string, backend api.Backend, consensus consensusAPI.Backend) {
```

Comment:
The name argument appears to always be the empty string, unless I'm mistaken.

Reply:
You are mistaken, it can also be the string `"client"` for the client backend (see `node_test.go`). It is required as there is no way to deregister runtimes so each backend test needs its own seed.